### PR TITLE
Simplify ExistsFilter declaration

### PR DESCRIPTION
**Pull request by antograssiot**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        | 

This PR aims to provide an alternative to https://github.com/api-platform/core/pull/2640
As i'm working on supporting name converters in filters, adding another optional argument is way harder now, so i went with what have been done in #1844

### CODE REVIEW

File: src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php [hunk 1]
```diff
@@ -44,33 +43,9 @@ class ExistsFilter extends AbstractContextAwareFilter implements ExistsFilterInt
 
     /**
      * @param RequestStack|null $requestStack No prefix to prevent autowiring of this deprecated property
-     * @param mixed|null        $properties
      */
-    public function __construct(ManagerRegistry $managerRegistry, $requestStack = null, LoggerInterface $logger = null, /* string $existsParameterName = self::QUERY_PARAMETER_KEY, array*/ $properties = null)
+    public function __construct(ManagerRegistry $managerRegistry, $requestStack = null, LoggerInterface $logger = null, array $properties = null, string $existsParameterName = self::QUERY_PARAMETER_KEY)
```
Comment (teohhanhui) [inline src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php]: The same strategy from #2664 should be applied here.

Comment (antograssiot) [reply src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php]: is it really worth it ? This argument is truly optional

Comment (antograssiot) [reply src/Bridge/Doctrine/Orm/Filter/ExistsFilter.php]: This comment was about changing the typehint of `$requestStack` as explained off, it has been done

**Comment by antograssiot** [comment]
Failing tests are likely not related and were fixed by @teohhanhui in https://github.com/api-platform/core/pull/2744/files#diff-762fac8fef8061a4affa4d35df9c45c7R846
This will need to get merged before and then 2.4 in master to avoid a long term broken testsuite

**Comment by teohhanhui** [comment]
@antograssiot Needs a rebase, then everything should be green? :smile:

**Comment by antograssiot** [comment]
here you are, all green 😄

**Comment by teohhanhui** [comment]
Thanks @antograssiot! :tada:

**Pull request merged**